src/inmanta_module_factory/helpers/utils.py

```diff
@@ -24,12 +24,247 @@
 import tempfile
 
 import inmanta
+import inmanta.ast.type
 import inmanta.module
+import inmanta.parser.plyInmantaLex
 
 from inmanta_module_factory.helpers import const
 
 LOGGER = logging.getLogger(__name__)
 
+assert inmanta.parser.plyInmantaLex.t_ID.__doc__ is not None, "Invalid lexer specifications"
+"""
+If this assertion fails, this is likely a compatibility issue with core, this docstring
```

Comment:
```suggestion
If this assertion fails, this is likely a compatibility issue with inmanta-core, this docstring
```
If this is meant as a message to the end user?

Reply:
It is meant for whoever comes to investigate why this assertion failed (as it might one day)